wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -119,18 +134,17 @@ class KotlinGenerator private constructor(
    * [service] will be code generated.
    */
   fun generateService(service: Service, rpc: Rpc? = null): TypeSpec {
-    val (interfaceName, rpcs) =
-        if (rpc == null) generatedServiceName(service) to service.rpcs()
-        else generatedServiceRpcName(service, rpc) to listOf(rpc)
-
-    val superinterface = com.squareup.wire.Service::class.java
+    val serviceName = generatedServiceName(service, rpc)
+    val builder = TypeSpec.interfaceBuilder(serviceName)
+        .addSuperinterface(com.squareup.wire.Service::class.java)
+
+    val rpcs = if (rpc == null) service.rpcs() else listOf(rpc)
+    for (it in rpcs) {
```

Comment:
skeptical of `it` as an explicit for loop variable name

Reply:
`rpcs.forEach {`?